packages/compass-connection-import-export/src/hooks/use-export.ts

```diff
@@ -47,9 +50,7 @@ export function useExportConnections(
     open: boolean;
     trackingProps?: Record<string, unknown>;
   },
-  exportConnections = ConnectionStorage.exportConnections.bind(
-    ConnectionStorage
-  )
+  exportConnections?: CompassConnectionStorage['exportConnections']
```

Comment:
If we're using this only as a test only fallback now, we should probably rename it similar to other props like that we have with a test prefix

Reply:
I was able to get rid of this entirely because we now use hooks to get the injected service.